src/go/rdctl/pkg/factoryreset/delete_data.go

```diff
@@ -218,32 +219,61 @@ func removeDockerCliPlugins(altAppHomePath string) error {
 }
 
 func removePathManagement(dotFiles []string) error {
+	const startTarget = `### MANAGED BY RANCHER DESKTOP START \(DO NOT EDIT\)`
+	const endTarget = `### MANAGED BY RANCHER DESKTOP END \(DO NOT EDIT\)`
+
+	// bash files etc. break if they contain \r's, so don't worry about them
```

Comment:
Wait, nevermind. I forgot that this code won't run on Windows.

But, that raises another concern. IMO it would be a good idea to do one of two things:
- add a comment to `removePathManagement` that says that it should not be called on Windows
- at the beginning of `removePathManagement`, return an error if running on Windows

Reply:
I think checking for Windows should be a separate issue.

Reply:
Ok, if you want to leave it as it is that is fine by me.